imagetest/test_suites/shapevalidation/setup.go

```diff
@@ -109,6 +119,9 @@ Familyloop:
 				continue Familyloop
 			}
 		}
+		if err := t.WaitForVMQuota(shape.quota); shape.quota != nil && err != nil {
```

Comment:
All the other cases you have the guarantee that you have quota, this is the only case you'd have to deal with exceptions:

This would look cleaner (and would remove the need of validating qa inside WaitForVMQuota()):

```
if shape.quota != nil {
  if err := t.WaitForVMQuota(shape.quota); err != nil {
    return err
  }
}
```

Reply:
Done